signalfx-tracing/signalfx-nodejs-tracing/hapi_pg/client.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+// Initialize the tracer and initiate auto-instrumentation
```

Comment:
This comment doesn't really match what happens in the code below, and ends up being confusing to the reader.